app/src/main/res/values/strings.xml

```diff
@@ -1,12 +1,18 @@
 <resources>
     <string name="app_name">FaceRecognition</string>
 
-    <string name="model_Facenet">"Facenet"</string>
-
     <string name="main_NoPermissions_hint">Brakujące kluczowe uprawnienia. Aplikacja zostanie zamknięta.</string>
 
+    <string name="default_model_name">Facenet</string>
```

Comment:
I disagree with adding "default_model_name" to strings file. Assuming that there is no Facenet model, application won't be able to work. I think it is better to treat first model from the models array as a default.